index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    
+    <link rel="stylesheet" href="./style.css">
+    
+    <title>Portfolio</title>
+
+</head>
+<body>
+    <div class="container">
```

Comment:
- [ ] [OPTIONAL] There's very small extra space on all sides from the body in your design, kindly fix this. Most likely this is caused by the default styles provided by the browser, so a simple `margin: 0` on the body would fix this. You can learn more about these default built in styles [here](https://stackoverflow.com/questions/11578819/css-reset-what-exactly-does-it-do).
- [ ] [OPTIONAL] Kindly try to remove this container `div` as it doesn't serve any purpose here.